static/js/components/CreateCommentForm.js

```diff
@@ -201,6 +220,49 @@ export const ReplyToCommentForm = connect(mapStateToProps, mapDispatchToProps)(
   }
 )
 
+export const EditCommentForm = connect(mapStateToProps, mapDispatchToProps)(
```

Comment:
Should we rename this file to `CommentForm` since we now have a second component other than the one the module is named after?

Reply:
yeah that's a good idea